builds/platina.json

```diff
@@ -1,12 +1,12 @@
 {
   "response": [
     {
-        "filename": "ColtOS-v6.2-platina-20200715-Official.zip",
-        "url": "https://sourceforge.net/projects/coltos/files/Platina/ColtOS-v6.2-platina-20200715-Official.zip/download",
-        "timestamp": 1594787957,
-        "md5sum": "4e111d7f6918ee44496737bf3aa8b0bc",
-        "size": 955751321,
-        "version": "v6.2",
+        "filename": "ColtOS-v6.3-platina-20200805-Official.zip",
+        "url": "https://sourceforge.net/projects/coltos/files/Platina/ColtOS-v6.3-platina-20200805-Official.zip/download",
+        "timestamp": 1596600415,
+        "md5sum": "04e19b3d90bd5e832cebd7c32c031b7a",
+        "size": 899815367,
+        "version": "v6.3",
```

Comment:
It should be 10.0
See: https://github.com/Colt-Enigma/platform_packages_apps_Updater/commit/ddbda95693df8bbae1a21577d87ea5e957afa95c
https://github.com/Colt-Enigma/platform_vendor_colt/commit/6716a8ae901f18858d28c18251d26683e65daaa4